src/wx/linux-shell-wrapper.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# Force Adwaita-dark theme for GTK2 and 3+, other themes can cause serious
+# issues:
+
+# For GTK2:
+if [ -e /usr/share/themes/Adwaita-dark/gtk-2.0/gtkrc ]; then
+	GTK2_RC_FILES=/usr/share/themes/Adwaita-dark/gtk-2.0/gtkrc
+fi
+
+# For GTK3+:
+GTK_THEME=Adwaita:dark
+
+# Turn off vsync:
+
+# for mesa drivers.
+vblank_mode=0
```

Comment:
This is ignoring our vsync menu option. Can you check if `vblank_mode=1` works on your config (since I have nvidia proprietary drivers).

It goes like this, if i remember correctly: `vblank_mode=`
```
0 - no vsync.

1 - default no vsync, but obey application's choice.

2 - vsync on, but obey application's choice. <== Here is our issue

3 - vsync forced on, but app can choose when to swap buffers(?).
```